proto/toolchain.bzl

```diff
@@ -108,23 +109,23 @@ def _rust_proto_toolchain_impl(ctx):
         edition = ctx.attr.edition,
     )
 
+# Default dependencies needed to compile protobuf stubs.
 PROTO_COMPILE_DEPS = [
     "@io_bazel_rules_rust//proto/raze:protobuf",
 ]
-"""Default dependencies needed to compile protobuf stubs."""
```

Comment:
Are these docstrings ever visible without viewing this source file?

Reply:
Or were they ever?

Reply:
Not to my knowledge. Buildifier flagged these as defects so I moved them to be normal comments.